examples/tests/sdk_tests/fs/basic.main.w

```diff
@@ -0,0 +1,30 @@
+bring fs;
+
+let filename = "hello.txt";
+let data = "Hello, Wing!";
+
+fs.writeFile(filename, data);
```

Comment:
All of these tests need to write to a tmp directory because we run these tests in AWS lambda where nothing outside of /tmp is writable.

(See here https://github.com/winglang/wing/actions/runs/6498033748/job/17648493122 where I ran the tests in AWS and it failed for fs)

Reply:
Thank you for reviewing and providing assistance!!